test/e2e/run_volume_test.go

```diff
@@ -30,6 +30,16 @@ var _ = Describe("Podman run with volumes", func() {
 		return strings.Fields(session.OutputToString())
 	}
 
+	//nolint:unparam
+	mountVolumeAndCheckDirectory := func(volName, volPath, expectedOwner, imgName string) {
+		check := podmanTest.Podman([]string{"run", "-v", fmt.Sprintf("%s:%s", volName, volPath), imgName, "ls", "-al", volPath})
+		check.WaitWithDefaultTimeout()
+		Expect(check).Should(ExitCleanly())
+		Expect(check.OutputToString()).To(MatchRegexp(`^drwxr-xr-x\.\s+[0-9]+\s+%s\s+%s\s*[0-9]\s+.+\s+\.$`, expectedOwner, expectedOwner))
+		// Should match lines of this format, but with arbitrary username. Permissions always fixed, directory always .
+		// drwxr-xr-x. 1 mheon mheon    0 May 22 09:49 .
```

Comment:
why use `ls -al` when the output is hard to match? either use `ls -n` to only get the ids or better use `stat -c %u:%g`
stat should be part of the regual alpine based image